src/components/Header/BaseHeaderNavigation.test.jsx

```diff
@@ -22,21 +24,15 @@ describe("<BaseHeaderNavigation />", () => {
     beforeEach(() => {
       render(
         <BaseHeaderNavigation testMode>
-          <ul>
-            <li>
-              <a href={"#"}>Nav item one</a>
-            </li>
-            <li>
-              <a href={"#"}>Nav item two</a>
-            </li>
-            <li>
-              <BaseHeaderMenu label="Nav item three">
-                <a href={"#"}>Sub item one</a>
-                <a href={"#"}>Sub item two</a>
-                <a href={"#"}>Sub item three</a>
-              </BaseHeaderMenu>
-            </li>
-          </ul>
+          <BaseHeaderMenuItem itemUrl="#">Nav item one</BaseHeaderMenuItem>
+          <BaseHeaderMenuItem itemUrl="#">Nav item two</BaseHeaderMenuItem>
+          <BaseHeaderMenuItem>
+            <BaseHeaderMenu label="Nav item three">
+              <a href={"#"}>Sub item one</a>
+              <a href={"#"}>Sub item two</a>
+              <a href={"#"}>Sub item three</a>
+            </BaseHeaderMenu>
+          </BaseHeaderMenuItem>
```

Comment:
`BaseHeaderNavigation` now restricts the component types it allows as its children. This was also incorrect HTML, as `BaseHeaderNavigation` generates a `ul` tag, so this basically had nested `ul` tags.